src/util/hash.rs

```diff
@@ -317,6 +323,9 @@ impl Sha256Sum {
         tmp.copy_from_slice(sha2_1.result().as_slice());
         Sha256Sum(tmp)
     }
+    pub fn empty() -> Sha256Sum {
+        Sha256Sum([0u8; 32])
```

Comment:
An empty Sha256 (the hash of no data) is not all 0's.

Reply:
Suggestion on alternative? `unanchored_tip` property is all 0's?

Reply:
`zero()` is fine.

Reply:
Done!